linkerd/app/inbound/src/policy.rs

```diff
@@ -44,7 +45,7 @@ pub enum DefaultPolicy {
 #[derive(Clone, Debug)]
 pub struct AllowPolicy {
     dst: OrigDstAddr,
-    server: watch::Receiver<ServerPolicy>,
+    server: Cached<watch::Receiver<ServerPolicy>>,
```

Comment:
This is a little unfortunate. Could we get away with doing something like `T: ToOwned<Owned = watch::Receiver<ServerPolicy>>`?

Reply:
Yeah...I thought about something like that. I think it would be `Borrow<watch::Receiver<ServerPolicy>>` or `AsRef<...>` instead of `ToOwned`, though — a `ToOwned` impl would have to clone the receiver out of the `Cached` handle, so the receiver could be held even while the cache access is considered over...